test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1126,6 +1127,81 @@ public void TensorFlowTransformCifarSavedModel()
             }
         }
 
+        public class InMemoryImage
+        {
```

Comment:
Is this the same class as the one in ImageTests.cs?
They both seem to be in the same project. Can one of them be eliminated?

Reply:
Yeah, they're the same class. But I wouldn't know how to share them across those tests without messing up the namespaces (one test is in Microsoft.ML.Tests and the other one in Microsoft.ML.Scenarios), so I just think it's better to keep them separate.

Reply:
You can simply add a using Microsoft.ML.Tests in the this file and reference the class.

---
In reply to: [414190280](https://github.com/dotnet/machinelearning/pull/5056#discussion_r414190280) [](ancestors = 414190280)

Reply:
Yes, but that would make accessible anything from Microsoft.ML.Tests on Microsoft.ML.Scenarios, and that wouldn't be right?

Reply:
Ok, I'll use "using InMemoryImage = Microsoft.ML.Tests.ImageTests.InMemoryImage;" inside the tensorflow tests :wink: